packages/edge-login-ui-rn/src/common/actions/LoginAction.js

```diff
@@ -185,6 +185,9 @@ export function userLoginWithPin (data: Object, backupKey?: string) {
             dispatch(dispatchActionWithData(Constants.OTP_ERROR, e))
             return
           }
+          if (e.message === 'Unexpected end of data') {
```

Comment:
Rather than duplicate this logic, I would prefer to see it gathered into some kind of "translateError" function.